src/components/Button/Button.styled.ts

```diff
@@ -0,0 +1,38 @@
+import { styled, css } from 'styled-components'
+import { COLOR_VARIANTS, FONT_VARIANTS } from './Button.types';
+
+const fontBasic = css`
+  font-family: Lato, sans-serif;
+  font-size: 18px;
+`
+
+export const ButtonWrapper = styled.div<{
+  bgcolorvariant: COLOR_VARIANTS,
+  fontcapvariant: FONT_VARIANTS
+}>`
+  button {
+    text-transform: ${({fontcapvariant}) => fontcapvariant === FONT_VARIANTS.STANDARD ? 'none' : 'uppercase'};
+    color: ${({bgcolorvariant}) => bgcolorvariant === COLOR_VARIANTS.GREEN ? '#ffffff' : '#000000'};
+    background: ${({bgcolorvariant}) => bgcolorvariant === COLOR_VARIANTS.GREEN ? '#6C7716' : '#fefefe'};
+  
+    &:active {
+      background: ${({bgcolorvariant}) => bgcolorvariant === COLOR_VARIANTS.GREEN ? '#7E8B1A' : '#eeeeee'};
+    }
+  }
+`;
+
+export const Anchor = styled.a`
+  ${fontBasic};
+`;
+
+export const ButtonBody = styled.button`
```

Comment:
I think all buttons in DEMO will look like this so in my opinion these props should be in GlobalStyles file

Reply:
But all buttons also will be this component so I'd leave them in component folder.